app/models/image_builder.rb

```diff
@@ -24,6 +24,28 @@ def build_image(dir, build, executor, tag_as_latest:, **args)
       end
     end
 
+    # store logins in a temp file and make it not accidentally added via `ADD .`
+    def local_docker_login
+      Dir.mktmpdir 'samson-tmp-docker-config' do |docker_config_folder|
+        # copy existing credentials
+        regular_config = File.join(ENV["DOCKER_CONFIG"] || File.expand_path("~/.docker"), "config.json")
```

Comment:
Could maybe add `DOCKER_CONFIG` to `env.example`?

Reply:
done